src/components/app/mixins.css

```diff
@@ -0,0 +1,65 @@
+@import './variablesV2.css';
+
+@define-mixin contentSmallest {
+  font-family: var(--content-font);
+  font-size: 11px;
```

Comment:
same here

Reply:
The idea of the mixins is to have the font stylized the same way in any place of the app, and these values shouldn't change based on the styleguide, so you should just use the mixin that fits your needs.